src/sections/Community/Handbook/leadership.js

```diff
@@ -206,8 +233,10 @@ const CommunityGuide = () => {
               Understands the workflow of the Issues and Pull Requests
             </label>
           </div>
+          <TocPagination />
```

Comment:
https://user-images.githubusercontent.com/85789734/143778820-6f3d23ed-7129-49f3-b374-4fb4ec274ddc.mp4

The last two links in the Navigation of the `Contributor Ladder` don't navigate to any part. Same for `Repository Overview` and `Projects` page.

Reply:
Also if you notice, on navigation the Heading of that particular section isn't visible on any page. Can you please take a look into that so that it navigates across sections perfectly?

Reply:
@debo about the navigation , i tried a lot to find the answer for the headings not visible.

Reply:
Ok people, I appreciate people might be in rush and everything but this is starting to become a little annoying. Could you please make sure you double check who you tag when replying to comments and such? Thanks  lot for your understanding.

Reply:
@debo didn't mean to tag you btw was tagging @debo19

Reply:
I know, but it's not the first time it happens on this repo, people fo for the first option... anyway, no hard feelings.

Reply:
@debo19 Their is the same issue in the docs.meshery too about the navigation that the header is not visible.

Reply:
@YashKamboj yeah, that it is actually an issue with id based navigation i don't think there's a fix for it.

Reply:
So @warunicorn19 is this pr good to merge???

Reply:
Yup, looks good to me.

Reply:
@YashKamboj yeah, I realized we had this problem earlier too. Good work on this though 🎉